.github/workflows/pre-release.yml

```diff
@@ -9,25 +9,8 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-defaults:
-  run:
-    shell: bash
-
 jobs:
-  release:
-    runs-on: ubuntu-latest
-    name: pre release
-
-    steps:
-      - uses: radcortez/project-metadata-action@master
-        name: retrieve project metadata
-        id: metadata
-        with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
-          metadata-file-path: '.github/project.yml'
-
-      - name: Validate version
-        if: contains(steps.metadata.outputs.current-version, 'SNAPSHOT')
-        run: |
-          echo '::error::Cannot release a SNAPSHOT version.'
-          exit 1
\ No newline at end of file
+  pre-release:
+    name: Pre-Release
+    uses: quarkiverse/.github/.github/workflows/pre-release.yml@main
```

Comment:
Any chance to depend on tag at some point?

Reply:
The idea is to depend on main in case we need to introduce changes to the release without requiring changes to all projects, so I'm not saying it will be always main, but it will likely be like this for now.

Reply:
I see, it makes sense, thanks for the clarifications